.github/workflows/codecov.yml

```diff
@@ -0,0 +1,62 @@
+name: code coverage
```

Comment:
Rather than this, I would like to upload coverage results after every test runs

Reply:
Meaning appending to the existing ci workflow? Does that work with matrix builds?

Reply:
it should, I have tried it in other projects :)

Reply:
Ok, I made what I think should be the change.